digitalocean-valkey-migration-issue.md

```
@@ -0,0 +1,78 @@
+## References
+- [DigitalOcean Valkey Announcement](https://www.digitalocean.com/blog/introducing-managed-valkey)
+- [Valkey Project](https://github.com/valkey-io/valkey)
+
+## Priority
+**High** - This needs to be addressed before DigitalOcean fully deprecates their Redis service.
+
+## Labels
+- infrastructure
+- breaking-change
+- migration
+- high-priority
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)